src/FlatSharp.Compiler/FlatSharpCompiler.cs

```diff
@@ -621,15 +622,20 @@ private static string CreateCSharp(
                 return csharp;
             });
         }
+        catch (FlatSharpCompilationException e)
+        {
+            csharp = e.CSharp;
```

Comment:
I think this is only really useful when code generation for reading bfbs fails, so should never be hit by an end user, but is useful for flatsharp developers

Reply:
Agree. Looks like you have a nullability error.